hw/mcu/stm/stm32_common/syscfg.yml

```diff
@@ -466,5 +466,16 @@ syscfg.defs:
             When enabled, OS_TICKS_PER_SEC should be one of 128, 256, 512, 1024.
         value: 0
 
+    STM32_TIMER_AUTO_OFF_COUNT:
+        description: >
+            Turn off hal timer if it is not used for a specific number of overflows.
+            HAL timers are used for high frequency time measurement or delays.
+            Due to high frequency nature of the timer, counter overflows interrupts
+            would interrupt sleep event if timer is not in use.
+            This value when set to value graters then 0 will all to turn off timer
+            after number of overflow events without user code asking for timer value
+            or scheduling event.
+        value: 0
```

Comment:
maybe add range here (since uint8_t is used), otherwise looks good to me

Reply:
done